lib/sorcery/model/submodules/jwt.rb

```diff
@@ -16,19 +18,25 @@ def self.included(base)
           end
 
           base.sorcery_config.instance_eval do
+            @defaults[:@jwt_header] = "Authorization"
             @defaults[:@jwt_algorithm] = "HS256"
             @defaults[:@session_expiry] = 60 * 60 * 24 * 7 * 2 # 2 weeks
 
             reset!
           end
 
+          base.sorcery_config.after_config << :validate_secret_defined
```

Comment:
Looks like this should be `validate_token_header_defined`, then I can merge. Thanks for the PR!

Reply:
👍 good eye...